terraform/jupyterlab-ibmcloud/README.md

```diff
@@ -0,0 +1,119 @@
+# Deploy a Jupyter Lab Workspace on IBM Cloud VPC 
+
+Deploy a Juptyer Lab on Ubuntu 18.04 ppc64le running in IBM Cloud VPC
```

Comment:
We should put another few sentences about what high-level use cases this enables for users that aren't necessarily familiar with Jupyter... i.e., a few segments from https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html#overview

Reply:
done